src/Elkuent/Builder.php

```diff
@@ -207,6 +215,56 @@ public function getFresh($columns = array())
     }
 
     /**
+     * Specify how to aggregate results
+     * Structured as an Elasticsearch aggregation: https://www.elastic.co/guide/en/elasticsearch/reference/1.5/search-aggregations.html
+     * 
+     * @param  array $aggregation PHP array representation of JSON aggregation object
+     * @return void
+     */
+    public function aggregateBy($aggregation) 
```

Comment:
@jasonjlock and I discussed this yesterday. Technically, it is not *really* elasticsearch raw input... Expected input is a PHP array representation of the JSON object that is supplied to the es REST api. If we do name it raw, I would include support for JSON input as well. Sound reasonable?